### DISCUSSION TITLE
GPG_TTY

### DISCUSSION THREAD

**Discussion by sibyfi** [xonsh/xonsh#5206] [Q&A]
I'm trying to export the GPG_TTY in xonsh but am running into an issue where it's not being assigned a full tty as bash would.

.bashrc:
export GPG_TTY='tty'

In my environment this gets expanded to:
~ $ env | grep GPG
GPG_TTY=/dev/ttys000

The equivelant in xonsh:

.xonshrc
$GPG_TTY = 'tty'

env | grep GPG
GPG_TTY=tty

How can i modify my .xonshrc so it behaves as it would in bash?

Update:  when I run source-bash it works fine.  I would love how to get this working without having to source my .bashrc though.

**Comment by Qyriad** [comment]
I'm not actually sure why the bash version *does* work, but in xonsh it would be `$GPG_TTY = $(tty).strip()`.
